src/zope/interface/common/tests/test_idatetime.py

```diff
@@ -38,9 +38,7 @@ def test_interfaces(self):
 
 
 def test_suite():
-    suite = unittest.TestSuite()
-    suite.addTest(unittest.makeSuite(TestDateTimeInterfaces))
-    return suite
+    return unittest.defaultTestLoader.loadTestsFromName(__name__)
```

Comment:
IMHO, it would be better just to delete `test_suite` and the `if __name__ == '__main__'' bits:  I thing 'zope.testrunner' DTRT if they are missing.